Source/EventParser.swift

```diff
@@ -46,12 +50,14 @@ class EventParser {
     private func processField(field: Substring, value: Substring) {
         switch field {
         case Constants.dataLabel:
-            if !data.isEmpty {
-                data.append(contentsOf: "\n")
-            }
```

Comment:
I believe this is a change in the spec that allows a data block to start with an arbitrary number of newlines by repeated empty `data` fields.